worker/Cargo.toml

```
@@ -57,6 +57,7 @@ nusamai-citygml = {git = "https://github.com/reearth/plateau-gis-converter", tag
 nusamai-geometry = {git = "https://github.com/reearth/plateau-gis-converter", tag = "v0.0.1-rc2", features = ["serde"]}
 nusamai-plateau = {git = "https://github.com/reearth/plateau-gis-converter", tag = "v0.0.1-rc2", features = ["serde"]}
 nusamai-projection = {git = "https://github.com/reearth/plateau-gis-converter", tag = "v0.0.1-rc2"}
+nusamai-gltf = {git = "https://github.com/reearth/plateau-gis-converter", branch = "exports-nusamai-gltf"}
```

We have imported this branch and release it.
https://github.com/reearth/plateau-gis-converter/releases/tag/v0.0.1-rc3
```suggestion
nusamai-gltf = {git = "https://github.com/reearth/plateau-gis-converter", tag="v0.0.1-rc3"}
```

---

use tag